commons/connector/fitbit/fitbit_intraday_heart_rate_variability.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitIntradayHeartRateVariability",
+  "doc": "Intra day heart rate data from fitbit device.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)." },
+    { "name": "dailyRmssd", "type": "float", "doc": "The Root Mean Square of Successive Differences (RMSSD) between heart beats. It measures short-term variability in the user’s heart rate in milliseconds (ms)."},
```

Comment:
is this daily or intraday?

Reply:
HeartRateVariability seems to be collected once daily during sleep.

[Endpoint docs](https://dev.fitbit.com/build/reference/web-api/intraday/get-hrv-intraday-by-date/)

> This endpoint returns the Heart Rate Variability (HRV) intraday data for a single date. HRV data applies specifically to a user’s “main sleep,” which is the longest single period of time asleep on a given date. It measures the HRV rate at various times and returns Root Mean Square of Successive Differences (rmssd), Low Frequency (LF), High Frequency (HF), and Coverage data for a given measurement. Rmssd measures short-term variability in your heart rate while asleep.

Reply:
Seems from the response you get multiple values for a single sleep period

```
{
  "hrv": [
    {
      "minutes": [
        {
          "minute": "2021-10-25T09:10:00.000",
          "value": {
            "rmssd": 26.617,
            "coverage": 0.935,
            "hf": 126.514,
            "lf": 471.897
          }
        },
        {
          "minute": "2021-10-25T09:15:00.000",
          "value": {
            "rmssd": 34.845,
            "coverage": 0.988,
            "hf": 344.342,
            "lf": 1422.947
          }
        },
        {
          "minute": "2021-10-25T09:20:00.000",
          "value": {
            "rmssd": 36.893,
            "coverage": 0.981,
            "hf": 328.704,
            "lf": 298.071
          }
        },
....
```

If that is the case, then might be better to call this property `rmssd` instead of `dailyRmssd`

Reply:
It's intraday, indeed. Interestingly, their own docs call it `dailyRmssd`: `https://dev.fitbit.com/build/reference/web-api/intraday/get-hrv-intraday-by-interval#Response` but then the example uses `rmssd`...